### ISSUE TITLE
Typo Keyboard access Failing examples

### ISSUE THREAD

**Issue by smh243** [Section508Coordinators/DHSA-TT-200-A#112]
If you haven't already, please review the [Guidelines for Contributing](contributing.md) in the main folder of this repository before submitting any issues.

## Issue Type (put an X between the brackets to indicate issue type)
- [ ] System Bug (the course doesn't function properly)
- [x ] Typo/grammar
- [ ] Incorrect content
- [ ] 508 issue (something is not accessible according to our own test process)
- [ ] Other, specify...[replace this text with your issue type description]

## Description
[Description of the issue]

## Location
https://training.section508testing.net/mod/book/view.php?id=1229&chapterid=2676
Lesson 1: Keyboard Access
Failing examples
Example 3
## Actual Behavior / Observation
Step 3 If an interactive element does not have keyboard access, determine if there is another keyboard>accessible method available on the page that provides the same functionality, e.g., one of two print methods provided is keyboard accessible.

Remove > between keyboard accessible.

**Comment by drewnielson** [comment]
Thanks! Fixed.
